### PR TITLE
Core working group call notes: November 1st, 2024

### PR DESCRIPTION
**Pull request by charles37**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- Tutorial happening in ~5 weeks
+- Expected attendees: Brad, Tyler, Anthony, and speaker
+- Hardware currently in San Diego
+- Total conference attendance ~130, tutorial attendance TBD
+- Action items:
+- Schedule planning call for next week
+- Verify nothing is broken
+- Polish existing content
+- Brad working on Non-Vausal story driver upgrade
```
Comment (ppannuto) [inline doc/wg/core/notes/core-notes-2024-11-01.md:38]:
```suggestion
- Brad working on nonvolatile storage driver upgrade
```

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- Switch to new streaming offer
```
Comment (ppannuto) [inline doc/wg/core/notes/core-notes-2024-11-01.md:39]:
```suggestion
- Switch to new streaming slices discussed earlier
```

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Resolve infrastructure build questions
+- Consider sustainability of tutorial materials
+
+### GCC/Build System Discussion
+
```
Comment (ppannuto) [inline doc/wg/core/notes/core-notes-2024-11-01.md:44]:
```suggestion

 - https://github.com/tock/libtock-c/pull/470
```

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- Debate about printing commands during build process
+- Discussion about whether to suppress command output with @ symbol
+- Concerns about consistency with rest of build system
+
+### CHERI Implementation Progress
+
+- Meta pointer PR renamed to capability pointer
+- Two parallel efforts for x86 external clemency:
```
Comment (ppannuto) [inline doc/wg/core/notes/core-notes-2024-11-01.md:52]: Should there be a new section here?

**Review by ppannuto — CHANGES_REQUESTED** [review]

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+# Tock Core WG meeting notes 2024-11-01
+
+## Attendees
+
+- Amit Levy
+- Leon Schuremann
```
Comment (lschuermann) [inline doc/wg/core/notes/core-notes-2024-11-01.md:6]:
```suggestion
- Leon Schuermann
```

It's a tough one. :smile:

File: doc/wg/core/notes/core-notes-2024-11-01.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+- https://github.com/tock/libtock-c/pull/470
+- Debate about printing commands during build process
+- Discussion about whether to suppress command output with @ symbol
+- Concerns about consistency with rest of build system
+
+### CHERI Implementation Progress
+
+- Meta pointer PR renamed to capability pointer
+- Two parallel efforts for x86 external clemency:
```
Comment (alevy) [inline doc/wg/core/notes/core-notes-2024-11-01.md:64]:
```suggestion
### x86 port progress

- Two parallel efforts for x86 external clemency:
```

**Review by alevy — APPROVED** [review]

**Review by ppannuto — APPROVED** [review]

**Pull request merged**